pages/common/deno.md

```diff
@@ -0,0 +1,25 @@
+# deno
+
+> A secure runtime for JavaScript and TypeScript
+> More information: <https://deno.land/>.
+
+- Run a JavaScript or TypeScript file:
+
+`deno file.ts`
```

Comment:
```suggestion
`deno {{path/to/file}}`
```

Reply:
While I understand the intent of this suggestion, I don't think it matches the format of other similar commands. For example, see: 

- `tldr node`
- `tldr python`
- `tldr rustc`

Reply:
You mean this should have the `.ts` extension? Keep in mind that JS / TS extension is not required for a file to be recognized as such.

Reply:
True @zdroid, but it can also be used as a hint. I suggest this:
```suggestion
`deno {{path/to/file.ts}}`
```

Reply:
That's good, too. My main concern was about path-to-file format, not the extension.